index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+<hr class="hr1">
+<div class ="midlle">
+   <div class="inline-block">
+       <div class="inline-block"><a href="file:///Users/Reema/code/wdi/homework/w02d01/portfolio.html"> My Qualifications   |  </a></div>
+       <div class="inline-block"> <a href="file:///Users/Reema/code/wdi/homework/w02d01/portfolio.html">  Portfolio     |</a></div>
```

Comment:
this link will only work on your computer you need to write the relative path not an absolute